docs/configuration/integrations/troubleshoot.md

```diff
@@ -0,0 +1,85 @@
+# Troubleshooting
+
+The sections below include information to help you troubleshoot issues that you may come across while configuring the integration of Codacy with your Git provider.
+
+If the information provided on this page is not enough to solve your issue, contact [support@codacy.com](mailto:support@codacy.com) providing all the information that you were able to collect while following the troubleshooting instructions.
+
+## GitHub Cloud and GitHub Enterprise authentication {id="github"}
+
+### 404 error
+
+While trying to authenticate on GitHub you get the following error message:
+
+![Invalid client id](images/github-invalid-client-id.png)
+
+This means that there is a mismatch in the Client ID that Codacy is using to authenticate on GitHub.
+
+To solve this issue:
+
+1.  Make sure that the value of `clientId` in your `values-production.yaml` file is the same as the Client ID of the [GitHub App that you created](github-app-create.md)
+2.  If the values were different, you need to re-execute the `helm upgrade` command as described for [GitHub Cloud](github-cloud.md) or [GitHub Enterprise](github-enterprise.md)
```

Comment:
I do not think this is what you need.
We had a task (or at least talked about it) to evaluate reload of configs.

I think the last I heard you would need to take down the namespace. Not sure if a better command was introduced (i know its possible)

cc\ @paulopontesm @h314to @joaocosta-codacy

Reply:
Indeed, now that you mention this, I'm not sure if a simple `helm upgrade` is enough to correctly force the reload of these configurations. :thinking: 

@heliocodacy has experience with this, maybe he can also help.

Reply:
If we change something on the values.yaml, it's recommended to use the flag --recreate-pods to ensure that the pods that need that configuration are recreated.

Reply:
Yes, now I remember that in the test with Caxaria he did use this extra flag every time it was necessary to update the settings (we can check this in minute 47:12 of the [recording](https://codacy.slack.com/files/UTFLDNDN2/F0124RZULFK/mk8s_install_caxaria_-_part_1.mp4), for example).

Reply:
Can we make that explicit?

Reply:
Yep, I'm actually adding this flag in each place where it is required.